tests/test_flows.py

```diff
@@ -531,6 +531,18 @@ def test_client_association_dummy(self):
         self.check_log(env.agents[0].radios[0],
                        r"Got client disallow request for {}".format(sta.mac))
 
+    def test_client_association_link_metrics(self):
+        ''' This test verifies that a MAUT with an associated STA responds to
+        an Associated STA Link Metrics Query message with an Associated STA Link Metrics
+        Response message containing an Associated STA Link Metrics TLV for the associated STA.'''
+
+        sta_mac = "11:11:33:44:55:66"
+        debug("Send link metrics query for unconnected STA")
+        env.controller.dev_send_1905(env.agents[0].mac, 0x800D,
+                                     tlv(0x95, 0x0006, '{sta_mac}'.format(sta_mac=sta_mac)))
```

Comment:
```suggestion
                                     tlv(0x95, 0x0006, '{}'.format(sta.mac)))
```
The `{var}` form is very useful in f-strings, but without them it's just too verbose IMO.